src/Trinity.VSExtension/Trinity.TSL.ManagedWrapper/Trinity.TSL.ManagedWrapper.vcxproj

```diff
@@ -15,6 +15,7 @@
     <RootNamespace>TrinityTSLParserManagedWrapper</RootNamespace>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <ProjectName>Trinity.TSL.ManagedWrapper</ProjectName>
+    <WindowsTargetPlatformVersion>10.0.18956.0</WindowsTargetPlatformVersion>
```

Comment:
please revert. I'm working on the compatibility issues. thanks.

Reply:
Done.